timetagger/_config.py

```diff
@@ -14,6 +22,12 @@ class Config:
       form "user1:hash1,user2:hash2" where each hash is a salted hash (BCrypt)
       of the password. Used in the default startup script ``__main__.py``.
       You can generate credentials with https://timetagger.app/cred.
+    * `proxy_auth_enabled (bool)`: enables authentication from a reverse proxy
+      (for example Authelia). Default "False".
```

Comment:
I think perhaps we can remove this item, because the presence of either/both of the below items already is a flag that the proxy is enabled. What do you think?

Reply:
Not sure. I agree that it is kind of redundant, how would you implement it? The config values are always filled with the default value, therefor if we check whether one of those is set in the __main__.py then they will always be.

Reply:
mmm, good point. Technically we could make the default of either empty and use that as the flag. But it feels a bit awkward somehow. Let's keep it explicit using `proxy_auth_enabled` 👍

Reply:
Good idea, I think it can't hurt to be explicit here :+1: